server/src/main/java/org/spine3/server/entity/EntityRepository.java

```diff
@@ -227,17 +228,24 @@ public I apply(@Nullable EntityId input) {
             }
         });
 
-        final ImmutableCollection<E> result = findBulk(domainIds);
+        final ImmutableCollection<E> result = findBulk(domainIds, fieldMask);
         return result;
     }
 
     private E toEntity(I id, EntityStorageRecord record) {
```

Comment:
if you mean `toEntity`, it can't be made static since it uses classes type parameters.